integration_tests/e2e/updateFunctionalityFull.cy.ts

```diff
@@ -23,236 +23,240 @@ context('Update functionality - Full flow', () => {
     cy.task('stubPlpPrisonListPageUi')
     cy.task('stubGetFrontEndComponents')
     cy.task('getPrisonerById')
-    cy.task('getCiagPlan')
     cy.task('getUserActiveCaseLoad')
     cy.task('stubVerifyToken', true)
     cy.task('getLearnerEducation')
     cy.task('getPrisonerById', 'A3260DZ')
-    cy.task('getCiagPlan', 'A3260DZ')
     cy.task('updateCiagPlan', 'A3260DZ')
     cy.signIn()
   })
 
-  it('Existing plan - Hoping to get work page', () => {
-    cy.visit('/plan/create/A3260DZ/hoping-to-get-work/update')
+  describe('Happy path scenarios', () => {
```

Comment:
This cypress test, and the next one in this diff, are the only 2 that need to load the CIAG Plan (Induction).
I've moved the use of the stub out of the main `beforeEach`, and grouped all of the existing tests into a "happy path" set of tests. The stub is then used it it's `beforeEach`

The reason I have done this is that I intend to write some sad path tests (eg: Induction not found, and Induction API throws an exception), but I'll do those in my next PR - this PR is big enough already! 😱